jetty-maven-plugin/src/it/jetty-cdi-run-forked/pom.xml

```diff
@@ -4,13 +4,18 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.eclipse.jetty.its</groupId>
+    <artifactId>it-parent-pom</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
   <groupId>org.eclipse.jetty.its.jetty-cdi-run-forked-mojo-it</groupId>
   <artifactId>jetty-weld-minimal</artifactId>
   <version>1.0-SNAPSHOT</version>
```

Comment:
The `version` is not necessary now that there is a parent, I would removed it from all children POMs - less duplication.

Reply:
I don't get the need of this one. There is no reason to have parent and child having same version. 
Please bear in mind it's ONLY test projects which are not deployed anywhere and versions will NEVER change!
this it parent pom is purely convenient to have common version in all TEST projects. Which is again not mandatory.